src/lucky/tags/specialty_tags.cr

```diff
@@ -56,4 +56,22 @@ module Lucky::SpecialtyTags
   def raw(string : String)
     view << string
   end
+
+  # Generates an escaped HTML `&nbsp;` entity for the number of times specified
+  # by `how_many`. By default it generates 1 space character.
+  #
+  # ```
+  # link "Home", to: Home::Index
+  # span do
+  #   space
+  #   text "|"
+  #   space
+  # end
+  # link "About", to: About::Index
+  # ```
+  # Would generate `<a href="/">Home</a><span>&nbsp;|&nbsp;</span><a href="/about">About</a>`
+  def space(how_many : Int32 = 1)
```

Comment:
Nice! I think the only change I think would be helpful is calling this method `nbsp`. That way it is clear it is a non breaking space and not just a space string or something

Reply:
but then my branch name won't make sense 😝